### ISSUE TITLE
[Bug] About page - Its really allign with center?

### ISSUE THREAD

**Issue by inson1** [NimaKhajehpour/GuessThatPokemon#30]
![Screenshot_2023-10-19-04-48-38-862_com.nima.guessthatpokemon.jpg](https://github.com/NimaKhajehpour/GuessThatPokemon/assets/75314629/07a99331-b1ee-4c23-aeac-ade9d2a3d4b9)

**Comment by NimaKhajehpour** [comment]
yes it should be aligned perfectly in center, there might be an issue in which i will look at it but most likely its an illusion in which the way components are put it seems like it is not centered although it actually is.
